packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -135,10 +138,11 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
 
   MouseCursor? _mouseCursor;
 
-  Future<void>? _gameLoaderFuture;
-
-  Future<void> get _gameLoaderFutureCache =>
-      _gameLoaderFuture ?? (_gameLoaderFuture = widget.game.onLoad());
+  Future<void> get _loaderFuture {
```

Comment:
I think you can have async getters no problem:
```
class A {
  Future<void> get foo async {
    await bar();
  }
  
  Future<void> bar() async {}
}
```
would that make this slightly simpler?

Reply:
I was thinking:
```
if (foo != null)
  await foo()
if (bar != null)
  await bar()
```
but maybe it's not better, not sure